tools/dev_scripts/ankaios-start

```diff
@@ -6,8 +6,38 @@ LOG_DIR="$(readlink -f "$SCRIPT_DIR/../../logs")"
 
 mkdir -p "$LOG_DIR"
 
-echo "Staring up Ankaios server"
-RUST_LOG=debug /workspaces/ankaios/target/x86_64-unknown-linux-musl/debug/ank-server 2> "$LOG_DIR/ank-server" &
+start_with_tls_enabled() {
+    sudo "$SCRIPT_DIR/../certs/create_certs.sh"
 
-echo "Staring up Ankaios agent"
-RUST_LOG=debug /workspaces/ankaios/target/x86_64-unknown-linux-musl/debug/ank-agent --name agent_A 2> "$LOG_DIR/ank-agent" &
+    echo "Staring up Ankaios server with TLS enabled"
+    ANKSERVER_CA_PEM=.certs/ca.pem \
+    ANKSERVER_CRT_PEM=.certs/server.pem \
+    ANKSERVER_KEY_PEM=.certs/server-key.pem \
+    RUST_LOG=debug /workspaces/ankaios/target/x86_64-unknown-linux-musl/debug/ank-server 2> "$LOG_DIR/ank-server" &
```

Comment:
The script does not work for me in the "secure mode" it cannot start the server. Here are the traces:
```
[2024-07-02T08:59:11Z DEBUG ank_server] Starting the Ankaios server with 
    	server address: '127.0.0.1:25551', 
    	startup config path: '[no config file provided]'
[2024-07-02T08:59:11Z INFO  ank_server::ankaios_server] No startup configuration provided -> waiting for new workloads from the CLI
[2024-07-02T08:59:11Z DEBUG ank_server::ankaios_server] Start listening to agents...
[2024-07-02T08:59:11Z ERROR] server error: 'Certificate error: 'The given certificate file ".certs/ca.pem" has incorrect permissions!''
```

Reply:
Then let's check together, why this does not work on your machine.

Reply:
fixed. please, check again